tests/unit/test_transformer.py

```diff
@@ -337,7 +337,7 @@ def test_ensure_last_transform_job_none(transformer):
     return_value=INIT_PARAMS,
 )
 def test_attach(prepare_init_params, transformer, sagemaker_session):
-    sagemaker_session.sagemaker_client.describe_transform_job = Mock(name="describe_transform_job")
+    sagemaker_session.describe_transform_job = Mock(job_name="describe_transform_job")
```

Comment:
is `Transformer` supposed to use the new method? I'm confused about how this unit test change relates to the code change.

Reply:
My bad, I was intended to change the integ test instead: I searched on the browser but opened the wrong test file on IDE.